### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_ScreenReader: There are more than 6 list items present in the drop down of column 2 yet they all are announced as 6 of 7.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4172]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: -A11yMAS;-A11yTCS;-A11ySev2;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG1.3.1;-Benchmark;-NVDA;-AILimited;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Add data in specify input file screen and tab to "next" button and press enter.
5.Press enter on next button.
6.Launch screen reader as NVDA and tab to drop down in column 2 and press enter on it , navigate to each item using down key arrow. 

**Actual:**
On navigating to the first item in the list it is announced as 1 of 6 the 6th item is announced as 6 of 7 and the 7th and rest of the items are announced as 6 of 7.

**Issue not observed with JAWS**

**Expected:**
On navigating to the first item the 1st item should be announced as 1 of (total number of items in the list) and thus the rest should follow.

**User Impact:**
The screen reader users will get the incorrect list numbers.

**Attachment:**
